README.md

```diff
@@ -25,6 +25,10 @@ __Adding social authentication to your application is a breeze__ and just requir
 
 See [https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/tree/dev/samples/Mvc.Client](https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/tree/dev/samples/Mvc.Client) for a complete sample __using MVC 6 and supporting multiple social providers__.
 
+## Contributing
```

Comment:
Yeah, having two sections for the same things looks weird. Listing the contributors is also a good idea :+1: